feature-libs/storefinder/components/store-finder-store/store-finder-store.component.ts

```diff
@@ -23,11 +24,19 @@ export class StoreFinderStoreComponent implements OnInit {
     private routingService: RoutingService
   ) {}
 
+  loadData: Subscription = combineLatest([
```

Comment:
I'd prefer to keep the component as clean as possible; we like to have the load/reload logic inside the facade layer or even lower if possible. You might want to look at other services where we've done similar things.